src/Writer/Extension/PodcastIndex/Feed.php

```diff
@@ -106,6 +110,189 @@ public function setPodcastIndexFunding(array $value): Feed
         return $this;
     }
 
+    /**
+     * Set feed license
+     *
+     * @param array $value [identifier: string, url: string]
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function setPodcastIndexLicense(array $value)
+    {
+        if (! isset($value['identifier'], $value['url'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: "license" must be an array containing the keys "identifier" (node value) and "url"'
+            );
+        }
+        if (! is_string($value['identifier'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: "identifier" of "license" must be of type string.'
+            );
+        }
+        if (! is_string($value['url']) || ! filter_var($value['url'], FILTER_VALIDATE_URL)) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: "url" of "license": must be a url starting with "http://" or "https://"'
+            );
+        }
+        $this->data['license'] = $value;
+        return $this;
+    }
+
+    /**
+     * Set feed location
+     *
+     * @param array $value [description: string, geo: string|null, osm: string|null]
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function setPodcastIndexLocation(array $value)
```

Comment:
Please add the return type to all newly added methods

```suggestion
    public function setPodcastIndexLocation(array $value): self
```